### ISSUE TITLE
User lacke privilege: Settings Tab

### ISSUE THREAD

**Issue by timeapen** [IBM-UrbanCode/Jenkins-Pipeline-UCD#2]
We're trying to make use of this pipeline step and encounter the following error.  Any ideas?

```
Started by user DevOps User
[Pipeline] node
Running on master in /var/jenkins_home/jobs/paolo - ucd deploy/workspace
[Pipeline] {
[Pipeline] step
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: Invalid http response code returned when acquiring UCD system configuration:400 Bad Request
Error processing command: User lacks privilege: Settings Tab
Finished: FAILURE
```

**Comment by jrbarto** [comment]
Yes, this is a known issue at the moment which we are working to fix. This problem came up as a result of an added check for UCD in maintenance mode, which requires elevated administrative privileges. For now, on the plugin page you can click the "Download past plugin versions" link to download the past version.

**Comment by timeapen** [comment]
Thanks.  I'll try that!

**Comment by nhmathis** [comment]
This issue has been resolved in Jenkins Pipeline version 2.2. 
The latest versions can be downloaded here: https://developer.ibm.com/urbancode/plugin/jenkins-2-0/
